src/ape/api/accounts.py

```diff
@@ -29,7 +29,7 @@ def __dir__(self) -> List[str]:
         Display methods to IPython on ``a.[TAB]`` tab completion.
 
         Returns:
-            list[str]: Method names that IPython uses for tab completion.
+            List[str]: Method names that IPython uses for tab completion.
```

Comment:
An example of switching to the `typing` module types instead.
I think this will make docs easier to work on and cause less confusions for contributors.